app/src/main/java/org/systers/mentorship/view/fragments/EditProfileFragment.kt

```diff
@@ -55,7 +55,7 @@ class EditProfileFragment: DialogFragment() {
                 }
             }
         })
-        dialog.window!!.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE)
+        dialog?.window!!.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE)
```

Comment:
What is the use of adding a '?' here?

Reply:
We need to to either assert using !! or safeguard using ? for a possibly null dialog object, that is why I used ?